docs/create_snapshot_schedule.md

```diff
@@ -0,0 +1,143 @@
+# Creating snapshot schedule #
+Following is an example to create a snapshot schedule for a volume of name volume1:
+Below are the options which can be passed while creating a snapshot schedule.
+
+- -o virtualCopyOf=x          This option is mandatory. x is the name of the volume for which snapshot schedule has to be created.
+- -o scheduleFrequency=x      This option is mandatory. x is the string that indicates the snapshot schedule frequency.
+                            This string will contain 5 values which are seperated by space. Example x can be replaced with "5 * * * *"
+                            First field in the string represents the number of minutes that are passed scheduled hour to execute the 
+                            scheduled task. Second field in the string indicated hour at which task needs to be executed. Third field in
+                            the string indicates day of the month on which scheduled task has to be executed. Fourth field in the string
+                            indicates month in which the task needs to be executed. Fifth field in the string indicates day of a week on
+                            which task should be executed. x has to be specified in double quotes. Valid values for these fields are:
+                            
+                              Field         Allowed Values
+                              -----         --------------
+                               minute        0-59
+                               hour          * or 0-23
+                               day-of-month  * or 1-31
+                               month         * or 1-12
+                               day-of-week   * or 0-6 (0 is Sunday)
+
+- -o scheduleName=x           This option is mandatory. x is a string which indicates name for the schedule on 3PAR.
+                              If *scheduleName=auto* is passed via docker volume create , then the schedule name is 
+                              generated automatically based on timestamp. This is relevant for using the scheduleName 
+                              in StorageClass in Kubernetes environment.
+- -o retentionHours=x         This option is not mandatory option. x is an integer, indicates number of hours this snapshot will be retained.
```

Comment:
retentionHours are not supported for creating a snapshot schedule. If this option is used while creating a snap schedule, error will be thrown.